src/microbe_stage/ProcessStatistics.cs

```diff
@@ -71,13 +66,8 @@ public void RemoveUnused()
         }
     }
 
-    public SingleProcessStatistics GetAndMarkUsed(BioProcess forProcess)
+    public SingleProcessStatistics GetAndMarkUsed(TweakedProcess forProcess)
     {
-#if DEBUG
```

Comment:
I think this debug code could be updated to check if the process inside the tweaked process is valid. That would keep this safety check (that only is in debug mode, so active for only programmers) in-place.